Makefile

```diff
@@ -53,9 +53,9 @@ remove-tmp:
 	rm -rf ./tmp
 	rm -rf ./.vagrant
 	rm -rf ./.minio.sys
-	rm -rf ./example/.terraform
-	rm -rf ./example/terraform.tfstate
-	rm -rf ./example/terraform.tfstate.backup
+	rm -rf ./example/**/.terraform
+	rm -rf ./example/**/terraform.tfstate
+	rm -rf ./example/**/terraform.tfstate.backup
```

Comment:
Why two asterisks? 🤔

Reply:
Goes through all subdirs https://stackoverflow.com/questions/8532929/two-asterisks-in-file-path

Reply:
Nice!